src/index.ts

```diff
@@ -18,21 +19,33 @@ export function defineTSConfig (tsconfig: TSConfig): TSConfig {
   return tsconfig
 }
 
-export async function readPackageJSON (id?: string, opts: ResolveOptions = {}): Promise<PackageJson> {
+const FileCache = new Map<string, Record<string, any>>()
+
+export async function readPackageJSON (id?: string, opts: ResolveOptions & ReadOptions = {}): Promise<PackageJson> {
   const resolvedPath = await resolvePackageJSON(id, opts)
+  if (opts.cache && FileCache.has(resolvedPath)) {
```

Comment:
I'm wondering if we could make it a private cache (by user) passed as an object.

Reply:
so opts.cache could either be a boolean or object?